neo/Wallets/NEP6/NEP6Wallet.cs

```diff
@@ -265,12 +276,15 @@ IEnumerable<Coin> GetCoinsInternal()
 
         public override IEnumerable<UInt256> GetTransactions()
         {
-            foreach (UInt256 hash in indexer.GetTransactions(accounts.Keys))
-                yield return hash;
-            lock (unconfirmed)
+            if (indexer != null)
```

Comment:
Note, GitHub's diff of GetTransactions is a little weird. Since GetTransactions is using yield return, I wrapped the existing function in an indexer null check. So if the indexer is null, this function implicitly returns an empty UInt256 enumerable.